server/container_create_linux.go

```diff
@@ -534,6 +537,15 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID, contai
 		}
 		specgen.SetLinuxCgroupsPath(cgPath)
 
+		if t, ok := kubeAnnotations[podTerminationGracePeriodLabel]; ok {
+			// currently only supported by systemd, see
+			// https://github.com/opencontainers/runc/pull/2224
+			if useSystemd {
+				specgen.AddAnnotation("org.systemd.property.TimeoutStopUSec",
```

Comment:
Can we use something like `TimeoutStopSec` to avoid the `t+"000000"` below?

Reply:
@saschagrunert we could, but then @giuseppe would need to add support for `Sec` suffix in crun, and I understand why he'd rather not. So I'd rather have it as is for now, for the sake of crun compatibility.

Reply:
Okay sounds good. :+1: